src/stats.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+ */
+
+#include "taisei.h"
+
+#include "config.h"
```

Comment:
Not needed

Reply:
Clarification: this comment only applies to the `#include "config.h"` line; `taisei.h` must be included by every file.